src/commands/cloudmanager/commerce/get-command.js

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommand = require('../../../base-command')
+const { initSdk, getProgramId } = require('../../../cloudmanager-helpers')
+const { cli } = require('cli-ux')
+const commonFlags = require('../../../common-flags')
+
+class GetCommandCommand extends BaseCommand {
+  async run () {
+    const { args, flags } = this.parse(GetCommandCommand)
+
+    const programId = getProgramId(flags)
+
+    const commandRetrievalMessage = `Getting Status for Command ID# ${args.commandId}`
+    const commandRetrievedMessage = `Status for Command ID# ${args.commandId}`
+
+    cli.action.start(commandRetrievalMessage)
+
+    const result = await this.getCommand(
+      programId,
+      args.environmentId,
+      args.commandId,
+    )
+
+    cli.action.start(commandRetrievedMessage)
+    cli.action.stop('\n')
+    const { id, type, command, startedBy, startedAt, completedAt, status } = result
+    cli.table([{ id, type, command, startedBy, startedAt, completedAt, status }], {
+      id: {
+        get: row => row.id,
+      },
+      type: {
+        get: row => row.type,
+      },
+      command: {
+        get: row => row.command,
+      },
+      'Started By': {
+        get: row => row.startedBy,
+      },
+      'Started At': {
+        get: row => row.startedAt,
```

Comment:
@thoughtassassin please use `formatTime` here. See https://github.com/adobe/aio-cli-plugin-cloudmanager/blob/main/src/cloudmanager-helpers.js#L185 and https://github.com/adobe/aio-cli-plugin-cloudmanager/blob/main/src/commands/cloudmanager/execution/get-step-details.js#L49 (as a point of reference).